cmake/install_helpers.cmake

```
@@ -44,66 +44,63 @@ endfunction()
 function(ginkgo_install_library name)
     ginkgo_add_install_rpath("${name}" "${ARGN}")
 
-    if (WIN32 OR CYGWIN)
-        # dll is considered as runtime
-        install(TARGETS "${name}"
-            EXPORT Ginkgo
-            LIBRARY DESTINATION "${CMAKE_INSTALL_FULL_LIBDIR}"
-            ARCHIVE DESTINATION "${CMAKE_INSTALL_FULL_LIBDIR}"
-            RUNTIME DESTINATION "${CMAKE_INSTALL_FULL_BINDIR}"
-            )
-    else ()
-        # install .so and .a files
-        install(TARGETS "${name}"
-            EXPORT Ginkgo
-            LIBRARY DESTINATION "${CMAKE_INSTALL_FULL_LIBDIR}"
-            ARCHIVE DESTINATION "${CMAKE_INSTALL_FULL_LIBDIR}"
+    install(TARGETS "${name}"
+        EXPORT Ginkgo
+        LIBRARY
+        DESTINATION "${CMAKE_INSTALL_FULL_LIBDIR}"
+        COMPONENT Runtime
+        RUNTIME
+        DESTINATION "${CMAKE_INSTALL_FULL_BINDIR}"
+        COMPONENT Runtime
```

I think this formatting would be better (with COMPONENT just below DESTINATION, github seems to display it badly): 
```suggestion
        RUNTIME DESTINATION "${CMAKE_INSTALL_FULL_BINDIR}"
                         COMPONENT Runtime
```